src/test/java/com/solvd/mainPage/SortItemsTest.java

```diff
@@ -0,0 +1,138 @@
+package com.solvd.mainPage;
+
+import com.solvd.base.SauceDemoBaseTest;
+import com.solvd.pages.common.MainPageBase;
+import com.solvd.utils.SortOption;
+import com.zebrunner.carina.utils.R;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+public class SortItemsTest extends SauceDemoBaseTest {
+
+    @Test
+    public void sortItemsByNameAscTest() {
+        SortOption sortOption = SortOption.NAME_ASC;
+
+        R.CONFIG.put("capabilities.allowInvisibleElements", "true");
+        MainPageBase mainPage = authUtils
+                .loginWithDefaultUser()
+                .clickSortBtn()
+                .clickSortBy(sortOption);
```

Comment:
I suggest to split this into multiple lines of the code
```suggestion
        MainPageBase mainPage = authUtils.loginWithDefaultUser()
        mainPage.clickSortBtn()
                .clickSortBy(sortOption);
```